apps/server/src/shared/repo/importuser/importuser.repo.integration.spec.ts

```diff
@@ -147,6 +147,17 @@ describe('ImportUserRepo', () => {
 			});
 		});
 
+		describe('byExisting', () => {
+			it('should exclude deleted importUsers', async () => {
+				const school = schoolFactory.build();
+				const importUser = importUserFactory.build({ school, deletedAt: new Date() });
+				await em.persistAndFlush([school, importUser]);
+				const [results, count] = await repo.findImportUsers(school);
+				expect(results).not.toContain(importUser);
+				expect(count).toEqual(0);
+			});
+		});
+
```

Comment:
consider a test, where a user is created, and even if the name is the same, it should not be found if that user was deleted